pkg/kubelet/dockershim/docker_container.go

```diff
@@ -131,6 +131,11 @@ func (ds *dockerService) CreateContainer(podSandboxID string, config *runtimeapi
 			OpenStdin: config.Stdin,
 			StdinOnce: config.StdinOnce,
 			Tty:       config.Tty,
+			// Disable Docker's health check until we officially support it
+			// (https://github.com/kubernetes/kubernetes/issues/25829).
+			Healthcheck: &dockercontainer.HealthConfig{
+				Test: []string{"NONE"},
+			},
```

Comment:
fwiw, this causes lots of warnings in docker log like: 

Dec 11 08:50:29 bpdehamkubn01 dockerd[14450]: time="2017-12-11T08:50:29.993217562+01:00" level=warning msg="Unknown healthcheck type 'NONE' (expected 'CMD') in container XXXXX"

is this really the cleanest way to tell docker to not use the healthcheck?

Reply:
@asacbp - that's what they are recommending, https://github.com/docker/engine-api/blob/master/types/container/config.go#L16 So maybe worth asking on the docker github issues?

Reply:
This has just been fixed in docker, but hasn't been included in any release yet. 
https://github.com/moby/moby/pull/35533